server/server/displays/__init__.py

```diff
@@ -65,3 +65,54 @@ def render(self):
 
         message = generic_message(ActionType.DisplayRender, payload)
         self.notify(message)
+
+
+class DisplayHandle2D(DisplayHandle):
```

Comment:
Am I right in the thinking this is a remote display handle, as it proxies the requests to the client? We might want to update the name to indicate this?